### DISCUSSION TITLE
Please let me know How Cookies effect in some modes: Workspaces, Normal Containers, Private Containers and WideVine

### DISCUSSION THREAD

**Discussion by Stop-Unsecure-ZeroInstall-Updating** [Floorp-Projects/Floorp#494] [Q&A]
# Hello Team,

## Thank you for the prompt answer last day.

1. I understood that Firefox and possibly forked ones can separate many accounts and cookies (and Mozilla VPN) inside one Firefox with a Multi-Account-Container. Then, we can also login using many accounts without any effects among the accounts in the different Workspaces where any accounts, cookies, etc., are tied with the different Workspaces completely like the above. 

2. I also understood Private windows are spread from any other normal windows from making them to automatic deletions. So, Private windows within the same Workspaces also never interfere with each other during the timings when some Private windows 
are alive?
 
3. I have been afraid that the open DRM is actually dominant by Google's WideVine, in the very last day, you kindly let me know the difficulties of using open something actually just Google implementations. Then, I think the last two releases have WideVine same as the original FireFox.
If it's inside the scope of the NDA, I do Not ask more. If you could write anything here, please let me know because also heard Google charged much. Do the policies of the giant Google-san change?

SIncerely yours,
away

**Comment by Ryosuke-Asano** [comment]
Workspaces's using shared cookies for normal browsing by default.

Workspace containers, introduced in 11.4.0, can limit this. This is because cookies can be separated by pre-assigning containers to workspaces.

Container tabs can separate cookies on a per-tab basis.

Private windows use different cookies for each window.

Widevine is closed source, so Firefox uses Google's, Floorp uses Google's as well, and DRM is expensive and difficult to obtain, so this problem will probably not be solved for a long time.
